### PR TITLE
Set min OCP to 4.13

### PR DESCRIPTION
**Pull request by skonto**
Fixes JIRA #

## Proposed Changes

- Follow up to https://github.com/openshift-knative/hack/pull/256.

### CODE REVIEW

File: test/upgrade/installation/openshift.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ var (
 	// adminAcks includes admin acks for cluster upgrades.
 	// See https://access.redhat.com/articles/6955381 for more details.
 	adminAcks = []string{
-		"ack-4.11-kube-1.25-api-removals-in-4.12",
+		"ack-4.12-kube-1.26-api-removals-in-4.13",
 		"ack-4.13-kube-1.27-api-removals-in-4.14",
```
Comment (skonto) [inline test/upgrade/installation/openshift.go:28]: We might need admin acks for 4.16 too, see what others [have](https://github.com/openshift/managed-cluster-config/tree/master/deploy/osd-cluster-acks/ocp).

File: hack/lib/certmanager.bash [hunk 2]
```diff
@@ -16,21 +16,7 @@ function deploy_certmanager_operator {
...
-      yq write - --doc 2 spec.channel tech-preview | \
-      oc apply -f - || return $?
-
-      deployment_namespace="openshift-cert-manager"
-  else
-    echo "Running on OpenShift ${openshift_version} which supports GA'ed cert-manager-operator"
-
-    oc apply -f "${certmanager_resources_dir}"/subscription.yaml || return $?
-  fi
+  oc apply -f "${certmanager_resources_dir}"/subscription.yaml || return $?
```
Comment (skonto) [inline hack/lib/certmanager.bash:19]: I guess certmanager is GA on 4.13+

**Comment by skonto** [comment]
cc @dsimansk

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**